### ISSUE TITLE
Make elements of auto tiling directly accessible

### ISSUE THREAD

**Issue by Zwelf** [teeuniverse/teeuniverse#175]
Since there are only a few elements in auto tiling, they could be made always accessible below the Map layer tile/Zone Layer tiles menu on the right side.
This could make mapping with this editor more intuitive, but if there would be more auto tiling elements, there would be a need for a scroll bar.

**Comment by necropotame** [comment]
I agree with you. It's really irritating to have to set the zones type or the style of a layer
